docs/RELEASE_NOTES.md

```diff
@@ -10,6 +10,9 @@ Migrations:
 - Send_before column to the `Message` table.
 - Creator_id column to the `Campaign` table.
 
+Heroku:
+
+  The Heroku deployment method has been migrated to the container runtime. When upgrading an existing Heroku app, these [migration steps](HOWTO_HEROKU_DEPLOY.md#upgrading-an-existing-heroku-app) will need to be performed.
```

Comment:
This should link to a fully qualified URL so that the migration steps for this release are always correct (otherwise this links to the current branch being viewed in GitHub).

`https://github.com/MoveOnOrg/Spoke/blob/33e2edcc455836f36eafd56d585430ac1ceda515/docs/HOWTO_HEROKU_DEPLOY.md#upgrading-an-existing-heroku-app`